src/diffusers/pipelines/wuerstchen/pipeline_wuerstchen_prior.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 The HuggingFace Team. All rights reserved.
+# Copyright 2023 The HuggingFace Team. All rights reserved. 
```

Comment:
```suggestion
# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Reply:
@kashif as this pr is alright,I sincerely request you to merge it .